pymchelper/run.py

```diff
@@ -57,6 +58,11 @@ def main(args=sys.argv[1:]):
 
     set_logger_level(parsed_args)
 
+    # check if output directory exists
+    output_dir = os.path.dirname(parsed_args.output)
```

Comment:
Output dir can also be None. It is also expected. In such case writers will automatically construct output filename.
I know, it should be documented.